coldfront/core/project/templates/project/project_join_list.html

```diff
@@ -9,6 +9,46 @@
 
 {% block content %}
 <h1>Join a Project</h1><hr>
+<h5>Pending Join Requests</h5>
+  {% if user.userprofile.access_agreement_signed_date is None %}
+    <p>First Review and Sign the cluster user agreement. Only then you can Join a cluster project and gain access to the cluster.</p>
+  {% elif join_requests is None %}
+    <div class="alert alert-info" role="alert">
+      <i class="fa fa-info-circle" aria-hidden="true"></i>
+        No pending requests.
+    </div>
+  {% else %}
+    <table class="table">
+      <thead>
+        <tr>
+          <th scope="col" class="text-nowrap">ID</th>
+          <th scope="col" class="text-nowrap">Name</th>
+          <th scope="col" class="text-nowrap">PIs</th>
+          <th scope="col">Title</th>
+          <th scope="col">Cluster</th>
+          <th scope="col" class="text-nowrap">Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for project in join_requests %}
+          <tr>
+            <td><a href="/project/{{project.id}}/">{{ project.id }}</a></td>
```

Comment:
You can use `href="{% url 'project-detail' project.id %}"` here, which will be resistant to changes to the URL path.